operations/patch_util.go

```diff
@@ -289,6 +293,53 @@ func (p *patchParams) loadProject(conf *ClientSettings) error {
 	return nil
 }
 
+func (p *patchParams) setLocalAliases(conf *ClientSettings) error {
+	if p.Alias != "" {
+		for i := range conf.Projects {
+			if conf.Projects[i].Name == p.Project {
+				if errStrs := model.ValidateProjectAliases(conf.Projects[i].LocalAliases, "Local Aliases"); len(errStrs) != 0 {
+					return errors.Errorf("validating local aliases: '%s'", errStrs)
+				}
+				for _, alias := range conf.Projects[i].LocalAliases {
+					if alias.Alias == p.Alias {
+						p.addAliasToPatchParams(alias)
+					} else {
```

Comment:
This if/else statement doesn't make sense; you're saying "if the first alias in the list doesn't match, return an error. Besides, we should only error if the alias has an invalid definition (like at 300) since aliases could be project-level.